website/content/docs/enterprise/index.mdx

```diff
@@ -8,22 +8,79 @@ description: >-
 
 # Consul Enterprise
 
-[Consul Enterprise](https://www.hashicorp.com/consul) features address the organizational complexities of collaboration, operations, scale and governance. It includes capabilities that improve failure resilience, read scalability, and managing access in accordance with organizational policies. It also adds support for application and service architectures across complex network topologies. If you have already purchased Consul Enterprise, please see the [licensing section](#licensing) below.
-
-Features include:
-
-- [Admin Partitions](/docs/enterprise/admin-partitions)
-- [Audit Logging](/docs/enterprise/audit-logging)
-- [Automated Backups](/docs/enterprise/backups)
-- [Automated Upgrades](/docs/enterprise/upgrades)
-- [Enhanced Read Scalability](/docs/enterprise/read-scale)
-- [Redundancy Zones](/docs/enterprise/redundancy)
-- [Advanced Federation for Complex Network Topologies](/docs/enterprise/federation)
-- [Network Segments](/docs/enterprise/network-segments)
-- [Namespaces](/docs/enterprise/namespaces)
-- [NIA with Terraform Enterprise](/docs/nia/enterprise)
-- [Sentinel](/docs/enterprise/sentinel)
-- [OIDC Auth Method](/docs/security/acl/auth-methods/oidc)
-
-These features are part of [Consul
-Enterprise](https://www.hashicorp.com/consul).
+[Consul Enterprise](https://www.hashicorp.com/products/consul/)
+features address the organizational complexities of collaboration, operations, scale, and governance
+and are [available in several forms of Consul Enterprise](#consul-enterprise-feature-availability).
+If you have purchased Consul Enterprise, refer to
+[how to gain access to enterprise features](#gain-access-to-enterprise-features).
+
+Enterprise features include:
+
+#### Multi-Tenancy
```

Comment:
Did we have a discussion about properly nesting headings? These should be ## . I still think it should be "Multi-tenancy" because compound words are considered a single word, but we're not consistent enough on this to enforce.